src/tasks/sep/038-U17-op-betanets/config.toml

```diff
@@ -0,0 +1,45 @@
+# Jovian Betanet on u17
+l2chains = [
+    {name = "jovian-beta-0", chainId = 420110019},
+    {name = "jovian-beta-1", chainId = 420110020}
+]
+
+# Addresses for this task can be found here: https://github.com/ethereum-optimism/devnets/pull/170/files
+fallbackAddressesJsonPath = "src/tasks/sep/038-U17-op-betanets/addresses.json"
+templateName = "OPCMUpgradeV500"
+allowOverwrite = ["SuperchainConfig", "SuperchainConfigProxyAdmin"]
+
+[[opcmUpgrades]]
+chainId = 420110019
+# Jovian prestate:
+absolutePrestate = "0x03caa1871bb9fe7f9b11217c245c16e4ded33367df5b3ccb2c6d0a847a217d1b"  # prestates."1.8.0-rc.4" https://www.notion.so/oplabs/U17-2a1f153ee162804f94d0c10d0a1e1258
+expectedValidationErrors = "PROXYA-10,SYSCON-140,DF-30,PDDG-DWETH-30,PDDG-130,PLDG-10"
+# PROXYA-10: ProxyAdmin owner is not set to the Standard PAO multisig
+# SYSCON-140: The SystemConfig.SuperchainConfig is not the standard one
+# DF-30: DisputeGameFactory owner is not set to the Standard PAO multisig
+# PDDG-DWETH-30: Permissioned dispute game delayed WETH owner mismatch
+# PDDG-130: The PDDG Challenger is not the standard one
+# PLDG-10: Permissionless dispute game implementation not found. Expected as it's a permissioned chain.
```

Comment:
Manually checked, looks good. Only validation errors are to do with non standard _roles_, and the VM errors we saw before have gone.